### PR TITLE
remove id_token at login route

### PR DESCRIPTION
**Pull request by jlaswell**

### CODE REVIEW

File: ui/src/app/modules/login/login.component.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export class LoginComponent {
   private loginError :string
 
   constructor(private router: Router, private route: ActivatedRoute) {
+    localStorage.removeItem('id_token');
```
Comment (kyleferguson) [inline ui/src/app/modules/login/login.component.ts]: Will this force you to login everytime you visit the app? IE login, go to another site in the same tab/window, then go back and have to login again?

Comment (jlaswell) [reply ui/src/app/modules/login/login.component.ts]: This is only called when we render the login page (so the login route). If you head back to somewhere like `/app/applications` you'll be fine.

Do you think that we should update this to check for an expiration and then redirect/`removeItem` accordingly?

Comment (kyleferguson) [reply ui/src/app/modules/login/login.component.ts]: Yea, my thought was, we check everytime the app is visited if there is a token and if it's valid (I think), and within that method if it errors out for any reason we should remove the token (currently I believe it just redirects and the expired or invalid token remains in localStorage)

Comment (jlaswell) [reply ui/src/app/modules/login/login.component.ts]: Ok then. That makes sense; I'll make that change.

**Review by larkmullins — APPROVED** [review]

**Pull request merged**